sensu-plugins-systemd.gemspec

```diff
@@ -30,17 +33,17 @@ Gem::Specification.new do |s| # rubocop:disable Metrics/BlockLength
   # s.test_files    = Dir['test/*.rb']
   s.test_files    = s.files.grep(%r{^(test|spec|features)/})
 
-  s.add_runtime_dependency 'sensu-plugin', '~> 1.2'
+  s.add_runtime_dependency 'sensu-plugin', '>= 2', '< 5'
```

Comment:
if we are gonna keep 2.x support I think we probably want to make sure its `>= 2.0.1` as there was a bugfix thats important that is brought in. I'd probably just remove it to keep only latest major version lib but fine leaving more support if you think that its important for users.